src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Set-AzsComputeQuota.ps1

```diff
@@ -162,7 +192,8 @@ function Set-AzsComputeQuota {
                 $GetTaskResult_params = @{
                     TaskResult = $TaskResult
                 }
-                Get-TaskResult @GetTaskResult_params
```

Comment:
Revert and replace with 
```
New-QuotaCustomObject -Quota (Get-TaskResult @GetTaskResult_params)
```

You don't need to do Write-Output

Reply:
done